test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -148,7 +97,7 @@ def addUserCreds(self, context):
         )
         squish.clickButton(squish.waitForObject(self.NEXT_BUTTON))
 
-    def finishSetup(self):
+    def nextStep(self):
```

Comment:
why change to nextStep?

Reply:
this function moves to the next dailog
`Server-(nextStep)->Cred-(nextStep)`